### PR TITLE
Fix install gcloud for hosted vs2017.

### PR DESCRIPTION
**Pull request by ILMTitan**
Fix for the error installing gcloud in the Hosted VS2017 VSTS build agents.

### CODE REVIEW

File: install-cloud-sdk-build-task/install-cloud-sdk-build-task.pssproj [hunk 1]
```diff
@@ -34,13 +34,13 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="install-cloud-sdk.ps1" />
+    <Compile Include="test\install-cloud-sdk.tests.ps1" />
```
Comment (ivannaranjo) [inline install-cloud-sdk-build-task/install-cloud-sdk-build-task.pssproj]: Why is it called `.tests.ps1`? It is installing the cloud SDK or testing the install? I know that you are in fact installing the SDK but this doesn't seem like the right name for this. Also, why in the tests directory?

**Review by ivannaranjo — CHANGES_REQUESTED** [review]
Just a quick question about the name of the files and folder, `test` seems to be the wrong name since it implies that these are tests not actual production code.

File: install-cloud-sdk-build-task/install-cloud-sdk.ps1 [hunk 2]
```diff
@@ -43,7 +43,7 @@ try {
             gcloud config set disable_usage_reporting $disableReportingString
         }
     } else {
-        $InstallerUri = "https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.zip"
+        $InstallerUri = "https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk-windows-bundled-python.zip"
```
Comment (ivannaranjo) [inline install-cloud-sdk-build-task/install-cloud-sdk.ps1:46]: Is this URL guaranteed to always be the latest bits? Should we fix it at a know version instead? This URL does not show up in the [versioned](https://cloud.google.com/sdk/downloads#versioned) download page, we should probably use one of those instead.

File: install-cloud-sdk-build-task/install-cloud-sdk-build-task.pssproj (hunk 1 above)
Comment (ILMTitan) [reply install-cloud-sdk-build-task/install-cloud-sdk-build-task.pssproj]: It is a Pester unit test file, but you are right that it does not actually test anything. I will remove it for now.

**Comment by ILMTitan** [comment]
The only real change is getting a zip file that contains a bundled python instance.

**Review by ivannaranjo — APPROVED** [review]

**Pull request merged**